libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.ts

```diff
@@ -1,33 +1,72 @@
-import { Component, Input, Output, EventEmitter, ViewEncapsulation, OnChanges, SimpleChanges } from '@angular/core';
+import { Component, Input, Output, EventEmitter, ViewEncapsulation, OnChanges, SimpleChanges, OnInit } from '@angular/core';
 import { Media } from '../thumbnail.component';
-
+import { ThumbnailDetailsComponent} from '../thumbnail-details/thumbnail-details.component';
+import { DialogService, RtlService } from '@fundamental-ngx/core';
 @Component({
     selector: 'fdp-thumbnail-image',
     templateUrl: './thumbnail-image.component.html',
     styleUrls: ['./thumbnail-image.component.scss'],
     encapsulation: ViewEncapsulation.None,
 })
-export class ThumbnailImageComponent implements OnChanges {
+export class ThumbnailImageComponent implements OnChanges, OnInit {
+
+    constructor(private _dialogService: DialogService,  private  _rtlService: RtlService) {}
 
     @Input()
     mediaList: Media[];
 
     @Input()
     isHorizontal = false;
 
+    @Input()
+    maxImages = 5;
+
+
     @Output()
     thumbnailClicked: EventEmitter<Media> = new EventEmitter();
 
+    ngOnInit(): void {
```

Comment:
same as before, typedoc comments for fields and methods, and lifecycle hooks can be `@hidden`

Reply:
modified.